tests/multi_output_models.py

```diff
@@ -0,0 +1,53 @@
+from typing import List
+
+import numpy as np
+
+from sklearn.utils.multiclass import type_of_target
+
+from scikeras.utils.transformers import ClassifierLabelEncoder
+from scikeras.wrappers import KerasClassifier
+
+
+class MultiLabelTransformer(ClassifierLabelEncoder):
+    def fit(self, X: np.ndarray) -> "MultiLabelTransformer":
+        self._target_type = type_of_target(X)
+        if self._target_type != "multilabel-indicator":
+            return super().fit(X)
+        # y = array([1, 1, 1, 0], [0, 0, 1, 1])
+        # each col will be processed as multiple binary classifications
+        self.n_outputs_ = self.n_outputs_expected_ = X.shape[1]
+        self.y_dtype_ = X.dtype
+        self.classes_ = [np.array([0, 1])] * X.shape[1]
+        self.n_classes_ = [2] * X.shape[1]
```

Comment:
Why are these attributes defined? I recognize they're used by `ClassifierLabelEncoder`. But would it be possible to get away with the following?

``` python
class MultiLabelTransformer(BaseEstimator, TransformerMixIn):
    def transform(...):
        ...
    def inverse_transform(...):
        ...

```

Reply:
They're used by `dynamic_classifier` in `tests/mlp_models.py`.